src/main/scala/uk/gov/nationalarchives/tdr/validation/schema/JsonSchemaValidators.scala

```diff
@@ -0,0 +1,35 @@
+package uk.gov.nationalarchives.tdr.validation.schema
+
+import com.networknt.schema._
+import uk.gov.nationalarchives.tdr.validation.schema.JsonSchemaDefinition.BASE_SCHEMA
+import uk.gov.nationalarchives.tdr.validation.schema.extensions.DaBeforeToday
+
+import scala.jdk.CollectionConverters._
+
+object JsonSchemaValidators {
+
+  private val validators: Map[JsonSchemaDefinition, JsonSchema] = Map(BASE_SCHEMA -> baseJsonSchemaValidator)
+
+  case class ValidationError(property: String, code: String)
```

Comment:
This case class doesn't appear to be used? Assuming awaiting how to define the errors?